### ISSUE TITLE
Literal languages are lost on repeated `#<<` calls

### ISSUE THREAD

**Issue by no-reply** [ActiveTriples/ActiveTriples#245]
We lose language values when setting using existing values. This affects `#<<`, and `#set` calls when a `Relation` is the argument.

```ruby
require 'rdf/vocab'

class Book
  include ActiveTriples::RDFSource  
  property :title, predicate: RDF::Vocab::DC.title  
end  

book = Book.new

book.title << RDF::Literal("Moomin", language: :en)
book.title << RDF::Literal("Mummi",  language: :fi)

book.dump :ttl
# => "\n [ <http://purl.org/dc/terms/title> \"Moomin\",\n      \"Mummi\"@fi] .\n"﻿⁠
```

**Comment by no-reply** [comment]
This apparently affects every version, at least since 0.7.x.

**Comment by gkellogg** [comment]
The vocabulary classes don't retain language. It keeps strings or symbols, which are interpreted according to the range of the property they use. Language has never been supported in the Vocabulary class definitions.

**Comment by tpendragon** [comment]
@gkellogg They don't, but that's good to know about RDF::Vocabulary.

**Comment by tpendragon** [comment]
Some chatter in Hydra, summarized as such: This PR breaks identifiable in the following way - https://github.com/ActiveTriples/ActiveTriples/compare/literal_fix_breaking_spec#diff-2c44efc8662822e940a63660d6b0fcc2R171

**Comment by no-reply** [comment]
Closed by #246.
